src/components/MdTable/MdTable.vue

```diff
@@ -90,8 +90,8 @@
       mdCard: Boolean,
       mdFixedHeader: Boolean,
       mdHeight: {
-        type: Number,
-        default: 400
+        type: String,
+        default: '400px'
```

Comment:
how about `type: [Number, String]`,

when type is Number, add `px`?

Reply:
I agree on that too. I this will be more flexible.